packages/core/test/helpers/ucanto.js

```diff
@@ -180,6 +181,7 @@ export async function getAggregatorServiceServer (serviceProvider, options = {})
           return Server.ok(pieceAddResponse).join(fx.link())
         }
       }),
+      // @ts-expect-error todo
```

Comment:
todo?

Reply:
this is the legacy API mock that will go away